src/App.jsx

```diff
@@ -19,7 +27,13 @@ function App() {
     <CustomRoutes noLayout>
       <Route path="/set-password" element={<SetPasswordPage/>} />
     </CustomRoutes>
-    <Resource name="profiles" list={ListGuesser} edit={EditGuesser}/>
+    <Resource name="profiles" list={ProfileList} edit={ProfileEdit} create={ProfileCreate} icon={Group} />
```

Comment:
https://marmelab.com/react-admin/Resource.html

I think we should definitely use the recordRepresentation argument here, at the least for Profiles and maybe for some others.

Reply:
did it for Profiles. The only other table we would need it is drives but not sure whats best there